code/modules/mob/living/carbon/human/human_recoil.dm

```diff
@@ -17,5 +17,7 @@
 		base_recoil -= 0.4 //less recoil if your a good shot
 	if(usr.stats.getPerk(PERK_SMARTLINK))
 		base_recoil -= 0.2 //Worse than kriosans but probably enough to matter
+		if(usr.stats.getPerk(PERK_STRONGARMS))
+			base_recoil = 0.3 //Just a little
```

Comment:
is this meant to be a setter and not subtraction like the other two perks in this family?

Reply:
Yeah this should've been a `-=`

Reply:
0.3 is higher than 0.2 when the other one you get from marshals/cargo for steep price I dunno about that